pyomo/repn/beta/matrix.py

```diff
@@ -348,12 +348,12 @@ def _get_bound(exp):
     )
 
 
-# class _LinearConstraintData(_ConstraintData,LinearCanonicalRepn):
+# class _LinearConstraintData(ConstraintData,LinearCanonicalRepn):
 #
 # This change breaks this class, but it's unclear whether this
 # is being used...
 #
-class _LinearConstraintData(_ConstraintData):
+class _LinearConstraintData(ConstraintData):
```

Comment:
This is in `beta`, and I am not entirely sure of its actual use "in the wild".  As a result, I elected to leave it "as is", figuring that if that ever moved out of beta it would probably be rewritten.